Spigot-Server-Patches/0004-MC-Utils.patch

```diff
@@ -1,16 +1,29 @@
-From 9e9204318a93e93baf42cab944f182a7d8c38d78 Mon Sep 17 00:00:00 2001
+From 6eedc7262e0065a783f81e33daeffaf2afad0440 Mon Sep 17 00:00:00 2001
 From: Aikar <aikar@aikar.co>
 Date: Mon, 28 Mar 2016 20:55:47 -0400
 Subject: [PATCH] MC Utils
 
 Collection of utils to help reduce NMS diff
 
+diff --git a/src/main/java/net/minecraft/server/Chunk.java b/src/main/java/net/minecraft/server/Chunk.java
+index 00fb20d..ceef4f4 100644
+--- a/src/main/java/net/minecraft/server/Chunk.java
++++ b/src/main/java/net/minecraft/server/Chunk.java
+@@ -676,7 +676,7 @@ public class Chunk {
+     }
+ 
+     @Nullable
+-    public TileEntity a(BlockPosition blockposition, Chunk.EnumTileEntityState chunk_enumtileentitystate) {
++    public final TileEntity getTileEntity(BlockPosition pos) { return a(pos, EnumTileEntityState.IMMEDIATE); } @Nullable public TileEntity a(BlockPosition blockposition, Chunk.EnumTileEntityState chunk_enumtileentitystate) { // Paper - OBFHELPER
```

Comment:
It would be nice to have it formatted like:
```java
    @Nullable public final TileEntity getTileEntity(BlockPosition pos) { return a(pos, EnumTileEntityState.IMMEDIATE); }  // Paper - OBFHELPER
    @Nullable
    public TileEntity a(BlockPosition blockposition, Chunk.EnumTileEntityState chunk_enumtileentitystate) {
```